Feature To Do List

santiagorodriguezbermudez opened the pull request:
## To Do List Implementation

### On this implementation we accomplished:
- [x] Implemented forms for adding projects and To-Do tasks to each project
- [x] Implemented logic for adding dynamically projects and To-Do to the local storage
- [x] Developed edit functionality to project and project
- [x] Added styles to project and todo list.

shloch reviewed (changes requested):
## Hello @santiagorodriguezbermudez , @nurislam-ab 

Project works really great as required by requirements...great


![todo](https://user-images.githubusercontent.com/40712459/92421119-e510c700-f16e-11ea-88b5-022176cb555d.gif)

Your code is also well organised..i like that

![todo3](https://user-images.githubusercontent.com/40712459/92420844-7b43ed80-f16d-11ea-8750-6e0e7653a16d.gif)

### [changes required]

PLease validate the submission of empty forms. As it is now, you can submit 'empty' `PROJECT` and `TODOS`

![todo2](https://user-images.githubusercontent.com/40712459/92421117-de824f80-f16e-11ea-9cfb-d4361dac8f9b.gif)


Happy coding

santiagorodriguezbermudez commented:
## Code Review # 1

### On this code review we changed the following:
- [x] Added validation for the empty title on project form and Todo Form
- [x] Added validation when updating the project title or To-Do title

stratospherique reviewed (approved):
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**
